cmd/call-api-endpoint/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/alecthomas/kong"
+
+	"github.com/edulinq/autograder/internal/api"
+	"github.com/edulinq/autograder/internal/api/core"
+	"github.com/edulinq/autograder/internal/cmd"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/log"
+)
+
+var args struct {
+	config.ConfigArgs
+	cmd.CommonOptions
+
+	Endpoint   string   `help:"Endpoint of the desired API." arg:""`
+	Parameters []string `help:"Parameter(s) for the endpoint" arg:"" optional:""`
+}
+
+func main() {
+	kong.Parse(&args,
+		kong.Description("Execute an API request to the specified endpoint. For more information on available API endpoints, see the API resource file at: ../resources/api.json"),
```

Comment:
Where are the user running this from?
(Keep that in mind while looking at this line.)